go.mod

```diff
@@ -42,18 +42,20 @@ require (
 	google.golang.org/grpc v1.67.3
 	k8s.io/api v0.32.3
 	k8s.io/apiextensions-apiserver v0.32.3
-	k8s.io/apimachinery v0.32.3
+	k8s.io/apimachinery v0.33.0
```

Comment:
Do we have an in-flight PR to update all the deps to 0.33? We probably want to do this in tandem.

Perhaps we hold this until the 1.33 updates have gone through

Reply:
Yup. We'll do the Go 1.24 and CR bump before. CR will include v1.33.0

Reply:
Looks like I have to do everything

Reply:
@sbueringer 
Sorry for the late reply.
Yeah, I thought this PR will be closed since the upgrading CR includes this patch.
#12191 completely include my changes on this PR. I close this, then would review #12191 👍

Reply:
Thank you very much!